<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>explore</class>
 <widget class="QDialog" name="explore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>731</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>World of Warcraft</string>
    </attribute>
    <widget class="QTextEdit" name="wow_text">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>471</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/wowlogo.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Classic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>The Burning Crusade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Wrath of the Lich King</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cataclysm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Mists of Pandaria</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>161</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Warlords of Draenor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LKLUG</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#000000;&quot;&gt;ABOUT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Diablo III</string>
    </attribute>
    <widget class="QTextEdit" name="d3_text">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>371</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/d3logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Starcraft II</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/sc2logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="sc2_text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>631</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Hearthstone</string>
    </attribute>
    <widget class="QTextEdit" name="hs_text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>631</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/hslogo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
